ui/src/main/webapp/src/app/misc/about.component.html

```diff
@@ -0,0 +1,94 @@
+<div class="about-page">
+
+    <div class="container surface-shaded gutter-top">
+        <div class="row">
+            <div class="col-md-12">
+                <div class="about">
+                    <div class="row">
+                        <div class="col-md-2 about-icon gutter-top hidden-sm hidden-xs">
+                            <img src="../../../img/rhamt-square-248.png" width="172" style="margin-top: -12px;">
```

Comment:
Why do you use this unofficial logo here? We have official one and we could use .svg format, which is scalable.